### PR TITLE
Create wmt12.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description

Created the WMT12 article to add details about the event, including the schedule and results. 

## Type of PR

- Creates the article _[WMT12]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

**Comment by cefoo** [comment]
Hi @liashahnazaryan! 
This is great! Thanks for the PR!
Before merging, could you include the WMT22 link in `events.md`?
Thanks!

**Comment by liashahnazaryan** [comment]
> Hi @liashahnazaryan! This is great! Thanks for the PR! Before merging, could you include the WMT22 link in `events.md`? Thanks!

Thanks for the review, @cefoo. I've added the link in `events.md` and `wmt.md`.

File: events/wmt12.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+- [Translation](https://www.statmt.org/wmt12/translation-task.html)
+- [Metrics](https://www.statmt.org/wmt12/metrics-task.html)
+- [Quality estimation](https://www.statmt.org/wmt12/quality-estimation-task.html)
+
+## Schedule
+
+### Day 1
+
+Thursday, 7 June, 2012
```
Comment (bittlingmayer) [inline events/wmt12.md:44]: Not sure we need to include the day of the week

> 7 June 2012

File: events/events.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ featured: true
 | 28 October | WPTP 2012 | San Diego, California |
 | 28 October-1 November | AMTA 2012 | San Diego, California |
```
Comment (bittlingmayer) [inline events/events.md:173]: This looks odd.

Probably we should add spaces around dashes in this scenario, and maybe in all scenarios

**Review by bittlingmayer — APPROVED** [review]
I made some changes, have a look.

**Pull request merged**